edge-util/src/Microsoft.Azure.Devices.Edge.Storage.RocksDb/DbStoreProvider.cs

```diff
@@ -71,6 +74,11 @@ public void RemoveDbStore(string partitionName)
             }
         }
 
+        public void Close()
+        {
+            // No-op.
```

Comment:
The reason behind adding a CloseAsync method is to execute the backup process as part of shutting down gracefully. 

We didn't want to put it in the Dispose() method because that is traditionally used to *just* dispose resources (such as unmanaged handles) instead of executing product code such as backup. Introductions of new bugs that prevent the execution of the Dispose() method of DbStoreProvider(s) are likely as the disposal execution chain might not be executed due to an upstream consumer not calling it explicitly.